ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestRunnable.java

```diff
@@ -391,6 +391,18 @@ void startAgentsOn(PerfTest perfTest, GrinderProperties grinderProperties, Singl
 	 * @param grinderProperties grinder properties
 	 * @param singleConsole     console to be used.
 	 */
+
+
+	private boolean ignoreStop(Config config, StopReason stopReason) {
```

Comment:
I think it's better to name it with positiveWay.
What about..
```
isValidStopReason(StopReason stopReason)
```

And I think the config is not necessary to be passed here. 
it's member variable in this object.

Reply:
oh that naming is pretty better! i'll rename it!

I didn't check that config is a member variable... was too hasty 😓...